BrytescoreObjCAPI/Example/BrytescoreObjcAPI/ViewController.m

```diff
@@ -0,0 +1,242 @@
+//
```

Comment:
Can you open up the main files that we interact with (this one and the `BrytescoreAPIMangager.m` file) in Atom and resave them? I think Atom by default will remove any trailing whitespace or will clean out lines that are all spaces.